src/core/Chip/Chip/Chip.tsx

```diff
@@ -1,11 +1,11 @@
-import React, { Component, ReactNode } from 'react';
+import React, { Component, forwardRef, ReactNode } from 'react';
 import classnames from 'classnames';
 import { default as styled } from 'styled-components';
 import { baseStyles } from './Chip.baseStyles';
-import { HtmlButton, HtmlSpan } from '../../reset';
-import { Icon } from '../Icon/Icon';
-import { logger } from '../../utils/logger';
-import { VisuallyHidden } from '../../components/Visually-hidden/Visually-hidden';
+import { HtmlButton, HtmlButtonProps, HtmlSpan } from '../../../reset';
+import { Icon } from '../../Icon/Icon';
+import { logger } from '../../../utils/logger';
+import { VisuallyHidden } from '../../../components/Visually-hidden/Visually-hidden';
```

Comment:
Just a small thing, but maybe ordering imports based on their folder structure like this:

```tsx
import { HtmlButton, HtmlButtonProps, HtmlSpan } from '../../../reset';
import { VisuallyHidden } from '../../../components/Visually-hidden/Visually-hidden';
import { logger } from '../../../utils/logger';
import { Icon } from '../../Icon/Icon';
import { baseStyles } from './Chip.baseStyles';
```

Reply:
These seems to be better now, not like as commented but close enough to be clearer.